integration_test/third_party_apps_data/applications/dcgm/metadata.yaml

```diff
@@ -27,7 +27,6 @@ platforms_to_skip:
   - centos-7
   - centos-8
   - debian-10
-  - debian-11
```

Comment:
We might not have enough GPU quota to run this through the nightly build. Let's leave it out for now and then we can verify it manually on other platforms after.

Reply:
Kept `debian-11`.